src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
@dentarg Yeap, it's only related to Betamax usage.
Without betamax testing everything is working perfectly with URLConnection or Apache HTTP clients